src/biogeophys/WaterStateType.F90

```diff
@@ -134,7 +135,12 @@ subroutine InitAllocate(this, bounds, tracer_vars)
     call AllocateVar1d(var = this%wa_col, name = 'wa_col', &
          container = tracer_vars, &
          bounds = bounds, subgrid_level = BOUNDS_SUBGRID_COLUMN)
-
+    if (use_fan) then
+       call AllocateVar1d(var = this%h2osoi_tend_tsl_col, name = 'h2osoi_tend_tsl_col', &
```

Comment:
My sense is that we don't need tracer versions of this. If so, then this should be moved out of WaterStateType into WaterStateBulkType. Then you'd use a standard allocate rather than this AllocateVar1d (following what's done for other variables in WaterStateBulkType). No other changes should be needed in other modules to accommodate this move.